egui/src/grid.rs

```diff
@@ -190,22 +325,7 @@ impl GridLayout {
         self.col = 0;
         self.row += 1;
 
-        if self.striped && self.row % 2 == 1 {
-            if let Some(height) = self.prev_state.row_height(self.row) {
-                // Paint background for coming row:
-                let size = Vec2::new(self.prev_state.full_width(self.spacing.x), height);
-                let rect = Rect::from_min_size(cursor.min, size);
-                let rect = rect.expand2(0.5 * self.spacing.y * Vec2::Y);
-                let rect = rect.expand2(2.0 * Vec2::X); // HACK: just looks better with some spacing on the sides
-
-                let color = if self.style.visuals.dark_mode {
-                    Rgba::from_white_alpha(0.0075)
-                } else {
-                    Rgba::from_black_alpha(0.075)
-                };
-                painter.rect_filled(rect, 2.0, color);
-            }
-        }
+        self.do_paint(cursor.min, painter);
```

Comment:
`do_paint` checks if `row == 0` which it never is here. Better to just call `paint_row` here to avoid confusion:

```suggestion
        self.paint_row(cursor, painter);
```

Reply:
`paint_row` only paints a single row. The `paint` method iterates through all the row pickers and paints for every case that fits. If you prefer, I can break `paint` into `paint_rows` and `paint_cols`, but I personally feel that it's okay to call a single `paint` method that handles everything.